ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -439,90 +487,163 @@ TorchCore::fillTensorDim (torch::autograd::Variable tensor_meta, tensor_dim dim)
 }
 
 /**
- * @brief	create object of TorchCore class.
- * @param	_model_path	: the path of the model
+ * @brief Free privateData and move on.
  */
-void *
-torch_core_new (const char *_model_path)
+static void
+torch_close (const GstTensorFilterProperties * prop, void **private_data)
 {
-  return new TorchCore (_model_path);
+  TorchCore *core = static_cast<TorchCore *>(*private_data);
+
+  g_assert (core);
+
+  delete core;
+
+  *private_data = NULL;
 }
 
 /**
- * @brief	delete the TorchCore class.
- * @param	torch	: the class object
+ * @brief Load pytorch modelfile
+ * @param prop property of tensor_filter instance
+ * @param private_data : pytorch plugin's private data
+ * @return 0 if successfully loaded. 1 if skipped (already loaded).
+ *        -1 if the object construction is failed.
+ *        -2 if the object initialization if failed
  */
-void
-torch_core_delete (void *torch)
+static gint
+torch_loadModelFile (const GstTensorFilterProperties * prop,
+    void **private_data)
 {
-  TorchCore *c = (TorchCore *) torch;
-  delete c;
+  TorchCore *core;
+  const gchar *model_path;
+
+  if (prop->num_models != 1)
+    return -1;
+
+  core = static_cast<TorchCore *>(*private_data);
```

Comment:
I think this needs indentation fix - `core = static_cast < TorchCore * >(*private_data);` - this is one of the changes if you run `indent` with our project indent format

Reply:
Yes, I will fix them after running `indent`